src/rp2_common/pico_bootrom/include/pico/bootrom_structs.h

```diff
@@ -0,0 +1,211 @@
+#include "boot/picobin.h"
```

Comment:
I'm really not a fan of exposing this header of structs. It implies that this is a public API, which it isn't

Can we try with just using memory offsets, and then we need to guard with the rom_version also.